.github/workflows/main.yml

```diff
@@ -3,6 +3,7 @@ name: Create gigs
 on:
   push:
     branches: [main, master]
+    paths: 'pages/**'
```

Comment:
This small change, ensures the main workflow only runs when files in the `pages` dir have changed